### ISSUE TITLE
`SphericalTensor.plot` with `radius=True` is not plotting with correct magnitude 

### ISSUE THREAD

**Issue by blondegeek** [e3nn/e3nn#75]
I'm re-running `e3nn_tutorial` and it seems that the `plot` method in `SphericalTensor` is not producing the correct magnitude of the signal (see attached example from `simple_tasks_and_symmetry.ipynb`). Could it be something is different between spherical harmonic conventions in `s2grid` and those of the `spherical_tensor.projection`?

<img width="461" alt="simple_tasks_and_symmetry_-_Jupyter_Notebook" src="https://user-images.githubusercontent.com/4080470/80164642-6dab1400-858e-11ea-9eaf-8a059642c1a2.png">

**Comment by blondegeek** [comment]
@mariogeiger Would you mind checking that I didn't mess anything up in `adjusted_projection`? https://github.com/e3nn/e3nn/blob/master/e3nn/spherical_tensor.py#L33

**Comment by blondegeek** [comment]
The output is off by a factor of sqrt(pi).

**Comment by mariogeiger** [comment]
In `ToS2Grid` I have a normalization such that the signal on the sphere is of variance 1.
```python
if normalization == 'component':
  n = math.sqrt(4 * math.pi) * torch.tensor([
    1 / math.sqrt(2 * l + 1)
    for l in range(lmax + 1)
  ]) / math.sqrt(lmax + 1)
if normalization == 'norm':
  n = math.sqrt(4 * math.pi) * torch.ones(lmax + 1) / math.sqrt(lmax + 1)
```

while `projection` assumes no normalization.
I will add as option `normalization='none'`
